features/roles.feature

```diff
@@ -16,34 +16,3 @@ Feature: Administering Roles
     And a person called "Susan Scientist"
     When I add a new "Chief scientific advisor" role named "Chief Scientific Advisor to the FCO" to the "Foreign Office"
     Then I should be able to appoint "Susan Scientist" to the new role
-
-  Scenario: Adding a new role with the editionable worldwide organisation feature flag enabled
-    Given the organisation "Foreign Office" exists
-    And a person called "Susan Scientist"
-    And the editionable worldwide organisation feature flag is enabled
-    Then I should not see the woldwide organisation input field
-    When I add a new "Chief scientific advisor" role named "Chief Scientific Advisor to the FCO" to the "Foreign Office"
-    Then I should be able to appoint "Susan Scientist" to the new role
```

Comment:
We've swapped this round, so roles are now assigned to editionable worldwide organisations, rather than worldwide organisations being assigned to roles.  The tests [here](https://github.com/alphagov/whitehall/blob/8eceacc2024becb728f2e14a3677f655d3e04085/features/editionable-worldwide-organisations.feature#L44-L48) should cover that.